### DISCUSSION TITLE
devspecs.dll

### DISCUSSION THREAD

**Discussion by RanjSharma** [open-watcom/open-watcom-v2#1351] [General]
I am using this dll in a 32bit windows environment and want to know if there is any documentation about it

**Comment by jmalak** [comment]
I am sorry, this DLL is not part of Open Watcom.

**Comment by RanjSharma** [comment]
Ok Thanks, but was it part of the old Watcom the dll header has "WATCOM C/C++32 Run-Time system. (c) Copyright by WATCOM International Corp. 1988-1994. All rights reserved.
"

**Reply by jmalak** [reply]
Please could you display contents(exported symbols) from DLL by Open Watcom wdump tool.
It could help us identify purpose and content.
I looked into Watcom 11 and 10 and there is not such file. It looks like part of some application builded by Watcom.
